src/client/components/VideoReserve/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+import { View } from '@instructure/ui-view';
+import { Heading } from '@instructure/ui-heading';
+import { Alert } from '@instructure/ui-alerts';
+import { Table } from '@instructure/ui-table';
+
+import { ltikPromise } from '../../services/ltik';
+import { PlayButton } from '../PlayButtonGroup/PlayButton';
+import { MediaView } from '../MediaView';
+
+export const VideoReserve = ({ course }) => {
+  VideoReserve.propTypes = {
+    course: PropTypes.object,
+  };
+
+  const [vidReserves, setVidReserves] = React.useState([]);
+  const getVideoRes = () => {
+    ltikPromise.then(ltik => {
+      axios.get(`/api/medias/videores?ltik=${ltik}`).then(res => {
+        setVidReserves(res.data);
+      });
+    });
+  };
+  React.useEffect(getVideoRes, []);
+
+  const [isOnCampusIP, setIsOnCampusIP] = React.useState(false);
+  const getOnCampusStatus = () => {
+    ltikPromise.then(ltik => {
+      axios.get(`/api/isoncampus?ltik=${ltik}`).then(res => {
+        setIsOnCampusIP(res.data);
+      });
+    });
+  };
+  React.useEffect(getOnCampusStatus);
```

Comment:
Will it be better if you do this within the 'getContext' route in app.js? It is a context that can be used by other components as well, and putting it in context makes it 1 less http request.